diff --git a/mesh/manager.c b/mesh/manager.c
index d70993e3b..2e604908d 100644
--- a/mesh/manager.c
+++ b/mesh/manager.c
@@ -367,7 +367,7 @@ static void prov_beacon_recv(void *user_data, struct mesh_io_recv_info *info,
 	builder = l_dbus_message_builder_new(msg);
 	l_dbus_message_builder_append_basic(builder, 'n', &rssi);
 	dbus_append_byte_array(builder, data + 2, len -2);
-	l_dbus_message_builder_enter_array(builder, "{sv}");
+	l_dbus_message_builder_enter_array(builder, "a{sv}");
 	/* TODO: populate with options when defined */
 	l_dbus_message_builder_leave_array(builder);
 	l_dbus_message_builder_finalize(builder);
