diff --git a/src/color_dialog/src/main.ts b/src/color_dialog/src/main.ts
index 9522499..7f290ae 100644
--- a/src/color_dialog/src/main.ts
+++ b/src/color_dialog/src/main.ts
@@ -1,4 +1,4 @@
-#!/usr/bin/gjs
+#!/usr/bin/env gjs
 
 imports.gi.versions.Gtk = '3.0';
 
diff --git a/src/floating_exceptions/src/main.ts b/src/floating_exceptions/src/main.ts
index 719c29b..7423d95 100644
--- a/src/floating_exceptions/src/main.ts
+++ b/src/floating_exceptions/src/main.ts
@@ -1,4 +1,4 @@
-#!/usr/bin/gjs
+#!/usr/bin/env gjs
 
 imports.gi.versions.Gtk = '3.0'
 
diff --git a/src/plugins/calc/main.js b/src/plugins/calc/main.js
index 3139f6d..713a38f 100755
--- a/src/plugins/calc/main.js
+++ b/src/plugins/calc/main.js
@@ -1,4 +1,4 @@
-#!/usr/bin/gjs
+#!/usr/bin/env gjs
 
 const { GLib, Gio } = imports.gi;
 
diff --git a/src/plugins/files/main.js b/src/plugins/files/main.js
index ba1a210..efa94e5 100755
--- a/src/plugins/files/main.js
+++ b/src/plugins/files/main.js
@@ -1,4 +1,4 @@
-#!/usr/bin/gjs
+#!/usr/bin/env gjs
 
 const { GLib, Gio } = imports.gi;
 
diff --git a/src/plugins/pulse/main.js b/src/plugins/pulse/main.js
index 1392ca6..df2e6d8 100755
--- a/src/plugins/pulse/main.js
+++ b/src/plugins/pulse/main.js
@@ -1,4 +1,4 @@
-#!/usr/bin/gjs
+#!/usr/bin/env gjs
 
 const { GLib, Gio } = imports.gi;
 
diff --git a/src/plugins/recent/main.js b/src/plugins/recent/main.js
index 91ad102..288f19d 100755
--- a/src/plugins/recent/main.js
+++ b/src/plugins/recent/main.js
@@ -1,4 +1,4 @@
-#!/usr/bin/gjs
+#!/usr/bin/env gjs
 
 const { GLib, Gio, Gtk } = imports.gi
 
diff --git a/src/plugins/terminal/main.js b/src/plugins/terminal/main.js
index 7446340..0564443 100755
--- a/src/plugins/terminal/main.js
+++ b/src/plugins/terminal/main.js
@@ -1,4 +1,4 @@
-#!/usr/bin/gjs
+#!/usr/bin/env gjs
 
 const { GLib, Gio } = imports.gi;
 
